tox.ini

```diff
@@ -2,7 +2,6 @@
 envlist =
     # flake8,
     py27,
-    py34,
```

Comment:
Why drop 3.4 support?  It remains supported by the Python community until May 2019 (5 years after the 3.4.0 final release).

Reply:
Fixed.